jwst/scripts/world_coords.py

```diff
@@ -71,33 +80,33 @@ def main(filenames, mode):
         is not in the image header
     """
     methods = {
-        'nrs_ifu': ifu_coords,
-        'nrs_fixedslit': compute_world_coordinates,
-        'nrs_msaspec': compute_world_coordinates,
-        'nrs_taconfirm': imaging_coords,
-        'nrs_brightobj': imaging_coords,
-        'nrs_bota': imaging_coords,
-        'nrs_tacq': imaging_coords,
-        'nrs_focus': imaging_coords,
-        'nrs_lamp': imaging_coords,
-        'nrs_mimf': imaging_coords,
-        'nrs_image': imaging_coords,
-        'nrs_confirm': imaging_coords,
-        'nrs_taslit': imaging_coords,
+        "nrs_ifu": ifu_coords,
+        "nrs_fixedslit": compute_world_coordinates,
+        "nrs_msaspec": compute_world_coordinates,
+        "nrs_taconfirm": imaging_coords,
+        "nrs_brightobj": imaging_coords,
+        "nrs_bota": imaging_coords,
+        "nrs_tacq": imaging_coords,
+        "nrs_focus": imaging_coords,
+        "nrs_lamp": imaging_coords,
+        "nrs_mimf": imaging_coords,
+        "nrs_image": imaging_coords,
+        "nrs_confirm": imaging_coords,
+        "nrs_taslit": imaging_coords,
     }
 
     ok = True
     outputs = build_output_files(filenames)
-    for (input_file, output_file) in zip(filenames, outputs):
-        if not os.path.exists(input_file):
-            warn_user(input_file, 'is not found')
+    for input_file, output_file in zip(filenames, outputs, strict=False):
```

Comment:
Do we ever expect these to be different lengths?  If not, I think strict should be set to True